subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -33,62 +33,51 @@ class AndroidSantaTrackerSmokeTest extends AbstractAndroidSantaTrackerSmokeTest
     }
 
     @UnsupportedWithConfigurationCache(iterationMatchers = AGP_4_0_ITERATION_MATCHER)
-    def "check deprecation warnings produced by building Santa Tracker Java (agp=#agpVersion)"() {
+    def "check deprecation warnings produced by building Santa Tracker (agp=#agpVersion)"() {
 
         given:
         AGP_VERSIONS.assumeCurrentJavaVersionIsSupportedBy(agpVersion)
 
         and:
         def checkoutDir = temporaryFolder.createDir("checkout")
-        setupCopyOfSantaTracker(checkoutDir, 'Java', agpVersion)
+        setupCopyOfSantaTracker(checkoutDir)
 
         when:
         def result = buildLocation(checkoutDir, agpVersion)
 
         then:
-        if (agpVersion.startsWith('4.0.2')) {
-            expectDeprecationWarnings(
-                    result,
-                    "The WorkerExecutor.submit() method has been deprecated. " +
-                            "This is scheduled to be removed in Gradle 8.0. Please use the noIsolation(), classLoaderIsolation() or processIsolation() method instead. " +
-                            "See https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_5.html#method_workerexecutor_submit_is_deprecated for more details."
-            )
-        } else if (agpVersion.startsWith('4.1')) {
-            expectDeprecationWarnings(result, "The WorkerExecutor.submit() method has been deprecated. " +
-                    "This is scheduled to be removed in Gradle 8.0. Please use the noIsolation(), classLoaderIsolation() or processIsolation() method instead. " +
-                    "See https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_5.html#method_workerexecutor_submit_is_deprecated for more details.")
-        } else {
-            expectNoDeprecationWarnings(result)
-        }
+        expectDeprecationWarnings(result, "The WorkerExecutor.submit() method has been deprecated. " +
```

Comment:
Isn't this deprecation warning gone with AGP 7.0?

Reply:
Looks like it's still a thing if the test passes, no? Is there something we need to do about this?

Reply:
We should have an issue for the problem on the AGP side. Is there already an issue?

Reply:
Or maybe this is a problem in the Kotlin plugin? Then we also should have an issue on the Kotlin plugin side.